tests/test_entities.py

```diff
@@ -1548,6 +1548,38 @@ def test_host_with_image(self):
                 # pylint:disable=no-member
                 self.assertIsNone(host.image)
 
+    def test_product_with_syncplan(self):
+        """Call :meth:`nailgun.entities.Product.read` for a product with syncplan assigned.
```

Comment:
Pls write a test for `search` coverage, not `read`, read is already covered in `ReadTestCase::test_product_with_sync_plan`.